specification/iotoperations/IoTOperations.Management/models/broker/brokers.tsp

```diff
@@ -56,18 +55,215 @@ model BrokerProperties {
   @doc("Memory profile of Broker.")
   memoryProfile?: BrokerMemoryProfile = BrokerMemoryProfile.Medium;
 
+  @visibility(Lifecycle.Read, Lifecycle.Create)
+  @doc("The persistence settings of the Broker.")
+  @added(Versions.`2025-07-01-preview`)
+  persistence?: BrokerPersistence;
+
   @visibility(Lifecycle.Read)
   @doc("The status of the last operation.")
   provisioningState?: ProvisioningState;
 }
 
+@doc("""
+  Disk persistence configuration.
+  
+  When persistence is enabled, certain items (non-performance-critical data) selected for persistence will reside only on disk. Below are the affected items: 
+  
+  - Retained messages will be stored on disk only. 
+  - WILL messages will be stored on disk only. 
+  - DSS key/value pairs will be stored on disk only, except for performance-critical items like timed locks, which remain in both disk and memory for improved performance.
+  
+  Optional. Everything is in-memory if not set. 
+  Note: if configured, all MQTT session states are written to disk.
+  """)
+@added(Versions.`2025-07-01-preview`)
+model BrokerPersistence {
+  @doc("""
+    Client sets the specified user property key/value in the CONNECT/SUBSCRIBE/PUBLISH 
+    Optional, default shown If customer specify a user property, it will work to enable persistence dynamically, in addition to the default user property "aio-persistence"
+    """)
+  dynamicSettings?: BrokerPersistenceDynamicSettings;
+
+  @doc("The max size of the message buffer on disk. If a PVC template is specified using persistentVolumeClaimSpec Then this size is used as the request and limit sizes of that template If a PVC template isn't specified Then local-path provisioner is requested with this size limit Required.")
+  maxSize: string;
+
+  @doc("""
+    Use the specified persistent volume claim template to mount a persistent volume Same object as in diskBackedMessageBuffer, but with a limitation that access modes field must be set to `ReadWriteOncePod`.
+    
+    If unset, a default PVC with default properties will be used. Among other things this PVC will use the cluster default storage class, which may or may not be using a local path provisioner. User is opting in to sub-optimal behavior if they leave this unset or set it without the storage class field, and their cluster default is not a local path class.
+    """)
+  persistentVolumeClaimSpec?: VolumeClaimSpec;
+
+  @doc("Controls which topic's retained messages should be persisted to disk.")
+  retain?: BrokerRetainMessages;
```

Comment:
nit: Should this be `BrokerRetainMessagesPolicy` to match the style of the below?